target/linux/ath79/nand/base-files/etc/board.d/01_leds

```diff
@@ -0,0 +1,19 @@
+#!/bin/sh
+
+. /lib/functions/uci-defaults.sh
+
+board_config_update
+
+board=$(board_name)
+boardname="${board##*,}"
+
+case "$board" in
+glinet,gl-ar300m-nand |\
+glinet,gl-ar300m-nor)
+	ucidef_set_led_netdev "lan" "LAN" "gl-ar300m:green:lan" "eth0"
+	;;
```

Comment:
Can you explain why you need to create this in nand subfolder? base-files are global for subtargets, so I would just have kept -nor in normal base-files and added -nand there?!

Reply:
ibid